internal/init/init.go

```diff
@@ -85,40 +86,31 @@ func (c *Client) NewCommand() *cobra.Command {
 			fmt.Println()
 			token := string(b)
 			c.Servicer.SetToken(token)
-			metalClient := c.Servicer.API(cmd)
-			c.UserService = metalClient.Users
-			c.ProjectService = metalClient.Projects
+			metalGoClient := c.Servicer.MetalAPI(cmd)
+			c.UserService = *metalGoClient.UsersApi
+			c.ProjectService = *metalGoClient.ProjectsApi
 
-			user, _, err := c.UserService.Current()
+			user, _, err := c.UserService.FindCurrentUser(context.Background()).Execute()
 			if err != nil {
 				return err
 			}
-			organization := user.DefaultOrganizationID
-			project := ""
-			if user.DefaultProjectID != nil {
-				project = *user.DefaultProjectID
-			}
+			organization := user.AdditionalProperties["default_organization_id"]
+
 			fmt.Printf("Organization ID [%s]: ", organization)
 
 			userOrg := ""
 			fmt.Scanln(&userOrg)
 			if userOrg == "" {
-				userOrg = organization
+				userOrg = fmt.Sprintf("%v", organization)
 			}
 
-			// Choose the first project in the preferred org
-			if project == "" {
-				project, err = getFirstProjectID(c.ProjectService, userOrg)
-				if err != nil {
-					return err
-				}
-			}
-			fmt.Printf("Project ID [%s]: ", project)
+			project := user.AdditionalProperties["default_project_id"]
+			fmt.Printf("Default Project ID [%s]: ", project)
```

Comment:
This prompt should not change; it's allowing the user to specify a project ID while also showing them what value will be used if they don't override it.
```suggestion
			fmt.Printf("Project ID [%s]: ", project)
```

Reply:
updated